<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPrefRhythm</class>
 <widget class="QWidget" name="DlgPrefRhythm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rhythm Detection Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>BPM Range</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Min:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="txtMinBpm">
        <property name="toolTip">
         <string/>
        </property>
        <property name="minimum">
         <number>50</number>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Max:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="txtMaxBpm">
        <property name="minimum">
         <number>50</number>
        </property>
        <property name="maximum">
         <number>300</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="bBeatsAndTempoEnabled">
     <property name="text">
      <string>Enable BPM and Beat Detection</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_20">
     <property name="text">
        <string>The beat is the smallest unit of rhythm. The tempo of the track is number of Beats Per Minute</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_21">
     <property name="text">
        <string>The downbeat is the first beat of a measure. The time signature specifies how many beats there are in a measure. </string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="bDownbeatsAndMeterEnabled">
     <property name="text">
      <string>Enable Downbeats and Time signature Detection</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_21">
     <property name="text">
        <string>A phrase is a combination of measures that has a complete musical sense. A section is a combination of phrases that represents a major strucral part of the track. </string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="bPhrasesAndSectionEnabled">
     <property name="text">
      <string>Enable Phrases and Sections Detection</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
